packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -163,8 +163,22 @@ export const ConfigurationPage = (props: ConfigurationPageProps): JSX.Element =>
   };
   /* @conditional-compile-remove(call-readiness) */
   const networkErrors = errorBarProps.activeErrorMessages.filter((message) => message.type === 'callNetworkQualityLow');
+
   /* @conditional-compile-remove(call-readiness) */
   const [isModalShowing, setIsModalShowing] = useState(true);
+
+  // When permission API is not available, we want to show screen saying checking for access (disappears on its own)
+  // then based on permission setting, we show permission denied or nothing
+  /* @conditional-compile-remove(call-readiness) */
+  const [checkPermissionModalShowing, setCheckPermissionModalShowing] = useState(true);
+  /* @conditional-compile-remove(call-readiness) */
+  const [permissionDeniedModalShowing, setPermissionDeniedModalShowing] = useState(false);
+  /* @conditional-compile-remove(call-readiness) */
+  setTimeout(() => {
```

Comment:
what is this timeout for?

Reply:
Oh I see this auto dismisses the checking. We want to keep the "checking" modal active until the user has either clicked accept or deny the system permission prompt:
![image](https://user-images.githubusercontent.com/2684369/201970586-a69863ad-0eab-404f-ac3f-bb00d51d5bf1.png)
So the checking modal should be showing and active whilst the systme prompt is displayed